tools/webdriver/webdriver/client.py

```diff
@@ -425,7 +429,26 @@ def send_command(self, method, url, body=None):
             an error.
         """
         response = self.transport.send(method, url, body)
-        value = response.body["value"]
+
+        if "value" in response.body:
+            value = response.body["value"]
+        elif "error" in response.body:
+            logger.warning("'value' key not found in response but 'error' key "
+                           "found in root. This is a compliance issue, but the tests are "
+                           "working around it.")
+            value = response.body
```

Comment:
After discussing on the IRC channel with the developer of chromedriver, we agreed that the payload should be correctly formatted.